src/main/java/com/deavensoft/paketomat/center/CenterServiceImpl.java

```diff
@@ -54,6 +62,49 @@ public void updateStatus(Long code, Status status) {
             }
         }
     }
+    public void payment(Long id, Paid paid)
+    {
+        List<Package> packages = centerRepository.findAll();
+        for (Package p : packages) {
+            if(p.getId().equals(id)){
+                p.setPaid(paid);
+                sendMailToUser(p.getUser().getEmail(),paid);
+                centerRepository.save(p);
+            }
+        }
```

Comment:
Does it make more sense to implement this method in a PackageServieImpl rather than here?

Reply:
Yes, but all logic for the package is in the CenterService.

Reply:
Ok, I didn't notice it. This must be refactored.  If you are working with packages, this must be PackageServiceImpl. 

Please rename 
- [ ] CenterController to PackageController
- [ ] CenterRepository to PackageRepository
- [ ] CenterService to PackageService
- [ ] CenterServiceImpl to PackageServiceImpl

Reply:
Done.